Doc/library/code.rst

```diff
@@ -112,7 +112,7 @@ Interactive Interpreter Objects
    with it.
 
 
-.. method:: InteractiveInterpreter.showsyntaxerror(filename=None)
+.. method:: InteractiveInterpreter.showsyntaxerror(filename=None, **kwargs)
```

Comment:
I don't really know what *kwargs* means here so I would appreciate that it's documented in a separate PR as well.